extensions/wikia/ArticleVideo/scripts/article-featured-video.js

```diff
@@ -84,22 +89,42 @@ require([
 				};
 
 			if (ooyalaAdSetProvider.canShowAds()) {
-				options.adSet = ooyalaAdSetProvider.get(1, correlator, {
-					contentSourceId: videoData.dfpContentSourceId,
-					videoId: videoId
-				});
 				options.replayAds = ooyalaAdSetProvider.adsCanBePlayedOnNextVideoViews();
+
+				if (a9 && context.bidders && context.bidders.a9Video) {
+					a9.waitForResponse()
+						.then(function () { return a9.getSlotParams('FEATURED'); })
+						.catch(function () { return {}; })
```

Comment:
Should we at least log it to the console? Ideally, we could send it to Kibana ;)

Reply:
I think there is no good place for storing bidder errors and no one want to check those errors :)
Main function of this catch is handling timeouts. As far as I understand, timeout for bidders should be logged anyway